docs/blog.md

```diff
@@ -1,42 +1,57 @@
 # Input Modality - `:focus`'ing on users
-Throughout the history of Web development, we're always trying to find ways to make the user experience better while maintaining a 'works for everyone' approach.  Early on, CSS had a concept called a [Media Sheet](http://www.w3.org/TR/2011/REC-CSS2-20110607/media.html#media-sheets) which allowed us to express that these styles over here are for print media, while those other ones over here were for users viewing on a screen.  
 
-Later, as the ways in which users viewed sites became increasingly more varied, it became evident that by just knowing a little bit more about the medium we were displaying on, we could do much better.  Thus, something more advanced was introduced in [CSS Media Queries Level 3](http://www.w3.org/TR/css3-mediaqueries/) which allowed authors to identify rules to target not just the overall _kind_ of media, but things _about_ the media, like it's size and whether or not it supported color.  From this, the modern Responsive Web movement was born and it's inarguably improved things.
+Web developers and web standards authors alike strive to live up to the promise of "universality" - the idea that the web _should_ be available to all. This concept drives many innovations in web technology, as well as being fundamentally built in to the philosophy of the open standards on which the web is based.
 
-Now, we find ourselves in a similar place, but from a different direction:  We're starting to realize that knowing how a particular user actually _interacts_ with content matters too.  To illustrate: Consider the humble focus ring.
+In order to achieve this, we find that having some carefully chosen __user context__ allows web developers to create more flexible and user experiences. We think it's time to expand our view of user context to include __modality__ - but before we flesh that out, let's recap what we can do today with user context. 
```

Comment:
Haha - "we" here meant the authors, and "user context" is a generic term (but IMO the key concept we're driving at throughout the post).